### DISCUSSION TITLE
Reality 分流给正常网站

### DISCUSSION THREAD

**Discussion by davuses** [XTLS/Xray-core#2239] [Q&A]
一台服务器用 xray 搭建 reality，同时服务器上有个正常网站，xray 占用了原来 nginx 监听的 443 端口，如何能把网站的访问流量分流给 nginx 呢？

**Comment by mclovin-2k** [comment]
"dest": "/dev/shm/nginx.socket"

**Reply by davuses** [reply]
谢谢回答，
```
"realitySettings": {
   "dest": "/dev/shm/nginx.socket"
}
```
是指这样配置吗？nginx那边该怎么配置呢？有更完整的模板吗？

**Comment by mclovin-2k** [comment]
> 谢谢回答，
> 
> ```
> "realitySettings": {
>    "dest": "/dev/shm/nginx.socket"
> }
> ```
> 
> 是指这样配置吗？nginx那边该怎么配置呢？有更完整的模板吗？

..........
server {
    listen unix:/dev/shm/nginx.socket    http2  ssl   proxy_protocol;
...........
